README.md

```diff
@@ -26,7 +26,11 @@ Demonstrate calling the method with an argument of "young prince".
 
 Write your code here:
 ```ruby
-# code here
+def offerRose(person)
+   person = "young prince"
+    puts "Would you take this rose and help out
+    an old beggar, #{person}?"
+  end
```

Comment:
Watch your indentation! I'd strongly recommend taking a bit just to look up and play with Atom's built-in indentation tools, like indenting multiple lines at once.